5.0/fr/0x17-V8-Authorization.md

```diff
@@ -0,0 +1,56 @@
+# V8 Autorisation
+
+## Objectif du contrôle
+
+L'autorisation garantit que l'accès est accordé uniquement aux utilisateurs autorisés (utilisateurs, serveurs et autres clients). Pour appliquer le principe du moindre privilège (POLP), les applications vérifiées doivent répondre aux exigences générales suivantes :
+
+* Règles d'autorisation des documents, y compris les facteurs décisionnels et les contextes environnementaux.
+* Les consommateurs ne devraient avoir accès qu'aux ressources autorisées par leurs droits définis.
+
+## V8.1 Documentation d'autorisation
+
+Une documentation complète des autorisations est essentielle pour garantir que les décisions de sécurité sont appliquées de manière cohérente, vérifiables et conformes aux politiques de l'organisation. Cela réduit le risque d'accès non autorisé en clarifiant les exigences de sécurité et en les rendant applicables aux développeurs, administrateurs et testeurs.
+
+| # | Description | Niveau |
+| :---: | :--- | :---: |
+| **8.1.1** | Vérifiez que la documentation d’autorisation définit des règles de restriction de l’accès au niveau de la fonction et aux données en fonction des autorisations du consommateur et des attributs des ressources. | 1 |
+| **8.1.2** | Vérifiez que la documentation d'autorisation définit les règles de restriction d'accès au niveau des champs (lecture et écriture) en fonction des autorisations des consommateurs et des attributs des ressources. Notez que ces règles peuvent dépendre d'autres valeurs d'attributs de l'objet de données concerné, telles que l'état ou le statut. | 2 |
+| **8.1.3** | Vérifiez que la documentation de l'application définit les attributs environnementaux et contextuels (y compris, mais sans s'y limiter, l'heure de la journée, l'emplacement de l'utilisateur, l'adresse IP ou l'appareil) qui sont utilisés dans l'application pour prendre des décisions de sécurité, y compris celles relatives à l'authentification et à l'autorisation. | 3 |
+| **8.1.4** | Vérifier que la documentation d'authentification et d'autorisation définit l'utilisation des facteurs environnementaux et contextuels dans la prise de décision, en plus des autorisations au niveau des fonctions, des données et des champs. Cela doit inclure les attributs évalués, les seuils de risque et les actions entreprises (par exemple, autorisation, contestation, refus, renforcement de l'authentification). | 3 |
+
+## V8.2 Conception d'autorisation générale
+
+La mise en œuvre de contrôles d’autorisation granulaires aux niveaux de la fonction, des données et des champs garantis que les consommateurs ne pourront accéder qu’à ce qui leur a été explicitement accordé.
+
+| # | Description | Niveau |
+| :---: | :--- | :---: |
+| **8.2.1** | Vérifiez que l’application garantit que l’accès au niveau des fonctions est limité aux consommateurs disposant d’autorisations explicites. | 1 |
```

Comment:
"accès au niveau des fonctions" I'm wondering if "function-level access" would be better translated as "accès basé sur la fonction" or something like that? (?)